### ISSUE TITLE
Format of the compass

### ISSUE THREAD

**Issue by SylvainCorlay** [jupyterlab/team-compass#22]
Hey, did you guys think of using github pages for the lab team compass.

We (just) started doing so for Voilà: https://voila-dashboards.github.io/ and plan on including more content as we go.

**Comment by saulshanabrook** [comment]
That looks really nice! Thanks for the pointer. I support making this, I don't have time to do it at the moment.

**Comment by choldgraf** [comment]
I am a big +1 on a non-github location for team compass material. It's useful to have an archive of "useful team information" that is easier to explore and search than a github repository.

**Comment by jasongrout** [comment]
I think it's a good idea for publishing info (meeting minutes, dates/times of meetings, etc.). I still like using github issues for discussion, though. For example, we couldn't collaborate on #19 very easily on a static website.

It does bring up the point, though: I think instead of our weekly meeting minutes being in issues, perhaps they should be actual files in this repo.

**Comment by choldgraf** [comment]
@jasongrout yep I agree. I think the right balance is to use a static site for "helpful information, team practices, minutes, etc" and to use issues for conversations or actionable items. That's what we do in jupyterhub land

**Comment by jasongrout** [comment]
For reference the jhub website is on readthedocs: https://jupyterhub-team-compass.readthedocs.io/en/latest/

**Comment by krassowski** [comment]
A note from a silent observer: the GitHub format works great for me because I can choose to "watch" and get notifications. In theory, I can also silence specific issues (by unsubscribing). I really want to stay up to date but have no time to visit a site regularly. I guess that if the static website would be hosted/generated from GitHub the notifications would keep working, but in other case staying up-to-date could be a slight problem (like would we need a newsletter then?).

**Comment by jasongrout** [comment]
> I can choose to "watch" and get notifications.

Thank you. That's a very good point.
